Source/Core/VideoBackends/Software/SWmain.cpp

```diff
@@ -41,7 +41,7 @@
 
 #define VSYNC_ENABLED 0
 
-static volatile u32 s_swapRequested = false;
+static std::atomic<bool> s_swapRequested;
 
 static volatile struct
```

Comment:
His comment was about ```static volatile struct { ... } s_beginFieldArgs```, not the ```s_swapRequested``` variable (or at least thats what I think your comment was referring to).